.github/workflows/daily.yml

```diff
@@ -86,8 +97,13 @@ jobs:
           # so we explicitly check Batteries as well here.
           lake env lean4checker/.lake/build/bin/lean4checker Batteries Mathlib
 
-      - name: Post success message on Zulip
-        if: success()
+      - name: Run mathlib_test_executable
+        id: mathlib-test
+        run: |
+          lake exe mathlib_test_executable
```

Comment:
The name of this executable feels slightly odd to me. I don't have an immediately better suggestion, though - and in any case, this is not so important.